sjb/config/test_cases/test_branch_openshift_ansible_logging.yml

```diff
@@ -108,6 +110,7 @@ extensions:
                          -e deployment_type=origin  \
                          -e etcd_data_dir="${ETCD_DATA_DIR}" \
                          -e openshift_pkg_version="$( cat ./ORIGIN_PKG_VERSION )"               \
+                         -e openshift_release="$( cat ./ORIGIN_SHORT_VERSION )"               \
```

Comment:
Is this an API change for the installer? Seems weird to need the version and release at the same time

Reply:
Also a little upsetting that this "release" is not the RPM release field

Reply:
Please see https://trello.com/c/Z83qZeDi/530-1-require-openshiftrelease

Reply:
It's a good point. We could check if the `openshift_pkg_version` is set and if so, detect the `openshift_release` from it. @sdodson WDYT?

Reply:
Yeah, that seems fine. If they've specified pkg_version we can set release to the first two members of that.